### DISCUSSION TITLE
Support pull request

### DISCUSSION THREAD

**Discussion by tunv11** [infinitered/ignite#2401] [Q&A] [answered]
i update feat i18n change.
i push code error
```
remote: Permission to infinitered/ignite.git denied to luvyouso.
fatal: unable to access 'https://github.com/infinitered/ignite.git/': The requested URL returned error: 403
```
thanks ignite red.

**Comment by frankcalise** [comment, marked as answer]
Hi @luvyouso, we do support pull requests, you just have to make it from a fork in your repo first I believe.

**Reply by tunv11** [reply]
https://github.com/infinitered/ignite/pull/2402
pls review.
